.github/workflows/build-pkg.yml

```diff
@@ -170,5 +194,5 @@ jobs:
         env:
```

Comment:
In addition to above: instead of `if: startsWith(github.ref, 'refs/tags/')` we could do `if: github.event_name == 'release'`